### DISCUSSION TITLE
svg converter not  support html tag?

### DISCUSSION THREAD

**Discussion by yugiking** [ElinamLLC/SharpVectors#181] [Q&A]
hi, 
I am using FileSvgConverter to convert my svg file to axml, my svg file content as below:

`    <svg viewBox="500 500 "> 
      <foreignObject>
        <text x="0" y="15" fill="red">I love SVG</text>
        </foreignObject>
    </svg> `

it has a foreignObject tag that include html elements , and the converted xaml as below:
`  <DrawingGroup x:Name="_SvgDrawingLayer">
    <DrawingGroup.ClipGeometry>
      <RectangleGeometry Rect="0,0,500,500" />
    </DrawingGroup.ClipGeometry>
  </DrawingGroup>`

so I can't get the html content in xaml, is it possible to support the html tags?

**Comment by paulushub** [comment]
There is no defined conversion process for the `foreignObject` tag. XAML will most likely not accept the HTML tags since it cannot process it. It might be possible to include it if you include the HTML schema in the XAML (never tried it).

Other possibility, you could convert the HTML to `FlowDocument`, but you cannot include it in a `DrawingGroup` object (never tried it). W3C Test Suites for WPF application contains the Microsoft source codes to convert from HTML to `FlowDocument`, and it is used by the Test Suite application to display the tests information included in HTML format.

After the conversion, you can query the SvgDocument, for the foreign tags as `SvgForeignObjectElement`, if you need to access the information there.

**Reply by yugiking** [reply]
Thanks for the reply, I will try your suggestion later.
